go/vt/vtgate/planbuilder/plan_test.go

```diff
@@ -501,7 +501,12 @@ func loadSchema(t testing.TB, filename string, setCollation bool) *vindexes.VSch
 			for _, table := range ks.Tables {
 				for i, col := range table.Columns {
 					if sqltypes.IsText(col.Type) {
-						table.Columns[i].CollationName = "latin1_swedish_ci"
+						switch col.Name.String() {
+						case "textcol2":
```

Comment:
In v18, the proto files don't contain the collation name, so I can't set it on `schema.json` as I did on `main`. This is the hack I did to get collations in.